config/kibana.yml

```diff
@@ -110,3 +110,4 @@
 # Specifies locale to be used for all localizable strings, dates and number formats.
 # Supported languages are the following: English - en , by default , Chinese - zh-CN .
 #i18n.locale: "en"
+vis_type_tagcloud.enabled: false
```

Comment:
Don't commit this :D

Reply:
oh ... )

Reply:
you can use `kibana.dev.yml` file to keep a specific configuration while switching between different branches. It's listed in `.gitignore` to avoid such cases when you commit something mistakenly.

Reply:
@mshustov, yes, thank you, sorry for this mistake.